docs/installation/box64.md

```diff
@@ -39,6 +47,169 @@ Box64 with Dynarec v0.3.3 7e5aa3e7 built on Dec 12 2024 22:01:08
 
 [Read more: What does the output of `box64 -v` mean? »](/docs/faq#what-does-the-output-of-box64--v-mean)
 
+## Cross-compiling/Installing for RV64 on x86_64
+
+However, compiling on RV64 platform will take more time due to their low CPU performance, so you can try to cross compiling it on your x86_64 PC.
+
+For example, you can compile Box64 on Ubuntu x86_64 and install it on Bianbu RV64.
+
+According to [Box64 cross-compiling document](https://github.com/ptitSeb/box64/blob/main/docs/COMPILE.md#cross-compiling):
+
+1. Set up the cross-compiler:
+   ```shell
+   sudo apt install gcc-riscv64-linux-gnu
+   ```
+   Some dependencies such as `binutils-riscv64-linux-gnu` will be installed automatically.
+   
+   You can check the installation with:
+   ```shell
+   riscv64-linux-gnu-gcc --version
+   ```
+   And you should get output like this:
+   ```
+   riscv64-linux-gnu-gcc (Ubuntu 13.3.0-6ubuntu2~24.04) 13.3.0
+   Copyright (C) 2023 Free Software Foundation, Inc.
+   This is free software; see the source for copying conditions.  There is NO
+   warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+   ```
+
+2. Cross-compile:
+   ```shell
+   cmake .. -D RV64=1 -D CMAKE_BUILD_TYPE=RelWithDebInfo -D CMAKE_C_COMPILER=riscv64-linux-gnu-gcc
+   make -j$(nproc)
+   ```
+   
+   Check the target file with `file box64`, you can find it is a `ELF 64-bit LSB executable, UCB RISC-V, RVC ...`, which means it is for RV64 platform.
+
+   About the CMake arguments:
+   - -D RV64=1: enable features related to the RISC-V 64-bit architecture.
+   - -D CMAKE_BUILD_TYPE=RelWithDebInfo: the generated code will be optimized while retaining debugging information for easier troubleshooting.
+   - -D CMAKE_C_COMPILER=riscv64-linux-gnu-gcc: specifies the C compiler to be used as riscv64-linux-gnu-gcc, the RISC-V cross-compiler
+
+3. Install:
+   
+   Instead of `make install`, you need to copy the executable and Box64’s shared libraries manually to your RISC-V device with tools like `scp`. For example:
+   ```shell
+   scp path/to/box64 user@riscv-device:/path/to/destination/
+   scp -r path/to/box64/x64lib user@riscv-device:/usr/lib/box64-x86_64-linux-gnu/
+   ```
+
+   If necessary, you might need to adjust file permissions on your RISC-V device to ensure the executable can be run:
+   ```shell
+   chmod +x /path/to/destination/box64
+   ```
+
+   Ensure that the library paths are part of the library search paths on your RISC-V device. You may also need to set the LD_LIBRARY_PATH environment variable:
+   ```shell
+   export LD_LIBRARY_PATH=/usr/lib/box64-x86_64-linux-gnu:$LD_LIBRARY_PATH
+   ```
+
+## Accelerate the build of Box64
+
+### Use mold
```

Comment:
There is a CMAKE option `USE_MOLD` in Box64, you can enable it and then use it with `mold -run make -j4`.